conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -13,7 +13,7 @@ docker info
 config=$(cat <<CONDARC
 
 channels:
-{% for channel in channels.get('sources', []) %}
+{% for channel in channels.get('sources', []) | sort %}
```

Comment:
When this was added, channel priority was not a thing. So sorting the channels likely would not have mattered much. However, now that channel priority is a thing. Sorting the channels instead of respecting the user specified order is a problem. As such am proposing we drop this channel sort in PR ( https://github.com/conda-forge/conda-smithy/pull/399 ).